drivers/ethernet/eth_sam_gmac.c

```diff
@@ -152,27 +150,25 @@ static inline void dcache_clean(u32_t addr, u32_t size)
 /* Get operations */
 static inline u32_t gmac_desc_get_w0(struct gmac_desc *desc)
 {
-	dcache_invalidate((u32_t)desc, sizeof(struct gmac_desc));
 	return desc->w0;
 }
 
 static inline u32_t gmac_desc_get_w1(struct gmac_desc *desc)
 {
-	dcache_invalidate((u32_t)desc, sizeof(struct gmac_desc));
 	return desc->w1;
 }
 
 /* Set operations */
 static inline void gmac_desc_set_w0(struct gmac_desc *desc, u32_t value)
 {
 	desc->w0 = value;
-	dcache_clean((u32_t)desc, sizeof(struct gmac_desc));
+	__DSB();
```

Comment:
I believe this line is redundant. In a few places in the driver code where we do care about the order of the memory accesses there is the DMB instruction. So DSB is no longer necessary. Unless there is another issue at hand I'm not aware of.